assemblies/karaf/pom.xml

```diff
@@ -158,8 +144,7 @@
         </plugin>
       <plugin>
         <groupId>org.apache.karaf.tooling</groupId>
-        <artifactId>features-maven-plugin</artifactId>
-        <version>${karaf.version}</version>
+        <artifactId>karaf-maven-plugin</artifactId>
```

Comment:
That is a modification that should have been sent with karaf 3.0.3 update commit, my bad